travis/licenses.golden

```diff
@@ -15254,7 +15254,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 LIBRARY: boringssl
 ORIGIN: ../../../third_party/boringssl/src/LICENSE
-TYPE: LicenseType.unknown
+TYPE: LicenseType.openssl
 FILE: ../../../third_party/boringssl/src/crypto/bn/asm/x86_64-gcc.c
 FILE: ../../../third_party/boringssl/src/crypto/err/asn1.errordata
 FILE: ../../../third_party/boringssl/src/crypto/err/bio.errordata
```

Comment:
Travis additionally noticed that when we're looking for a license via this new path, two files end up here instead of a block that has no license text to reproduce. Can you verify that the other files in the block that these two files used to be in are correctly being given classified as not needing a license reproduced in the license screen? (It's quite possible, but better safe than sorry.)